core/src/main/java/org/openconnectors/connect/PushSourceConnector.java

```diff
@@ -0,0 +1,14 @@
+package org.openconnectors.connect;
+
+import java.util.Collection;
+import java.util.function.Consumer;
+
+/**
+ * Source Connector meant be the start of a DAG pipeline, after initilization is configured with a collector
+ * and started with an execution instance
+ *
+ * @param <T>
+ */
+public interface PushSourceConnector<T> extends Connector {
```

Comment:
Push connectors will have a start/stop/pause mechanism

Reply:
We already have open/close. Why have more interfaces?

Reply:
open and close and analog to socket or db connection , start and stopping the data emission is different.

Reply:
ok we will have to define the examples for that layer also

Reply:
I agree. Which is why the BasicCopyTopology will be a good example of that.